presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcBatchPageSource.java

```diff
@@ -94,7 +94,7 @@ public OrcBatchPageSource(
             FileFormatDataSourceStats stats,
             RuntimeStats runtimeStats,
             // TODO avoid conversion; just pass a boolean array here
-            List<Boolean> isRowNumberList,
+            boolean[] isRowNumberList,
```

Comment:
This is not what the issue is about. Changing to an array doesn't help.

Reply:
Thanks for the response. I'll recheck this